.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -39,7 +39,7 @@
   fixed:
     E2E_PROVIDER: ocp
   mixed:
-    - DEPLOYER_CLIENT_VERSION: "4.16.1"
+    - DEPLOYER_CLIENT_VERSION: "4.16.15"
```

Comment:
`4.17` seems to have been released yesterday: https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html (note sure we want to immediately test this one though)

Reply:
Oh I missed this. I merge this one to get the e2e tests fixed and raise another one (in case there are issues that take longer to resolve)